packages/get-platform/src/getPlatform.ts

```diff
@@ -199,6 +199,12 @@ export async function getPlatform(): Promise<Platform> {
     return 'netbsd'
   }
   if (platform === 'linux' && arch === 'arm64') {
+    // 64 bit ARM
+    return `linux-arm64-openssl-${libssl}` as Platform
+  }
+
+  if (platform === 'linux' && arch === 'arm') {
+    // 32 bit ARM
     return `linux-arm-openssl-${libssl}` as Platform
```

Comment:
Is this still built or now bring your own engine file?

Reply:
This is for bring your own, just like nixos.

Reply:
So this just makes sure we can handle these platforms, but does not include any information if we have a engine files for it that can be downloaded?

Reply:
Yep, otherwise code will block the use of Prisma